server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const next = require('next');
+
+const dev = process.env.NODE_ENV !== 'production';
+const app = next({ dir: './src', dev });
+const handle = app.getRequestHandler();
+
+app
+  .prepare()
+  .then(() => {
+    const server = express();
+
+    server.get('/pageA', (req, res) => {
+      const actualPage = '/pageA/pageA';
+      app.render(req, res, actualPage);
+    });
+
+    server.get('/pageB', (req, res) => {
+      const actualPage = '/pageB/pageB';
+      app.render(req, res, actualPage);
+    });
```

Comment:
I think there's no need to add /page and /pageB here since the routes are the same (no friendly SEO), I would keep the server.js files but remoting from line 13 to 21

Reply:
But if we remove these lines, we can´t use _http://localhost:8080/pageB_ when call to server; we can only use _http://localhost:8080/pageB/pageB_.

¿Is the behavior that we want?

Reply:
I like your new folder structure, but I think we have to put pageA.tsx too,
```diff
|───components
│   ├───pageA
│   │       index.tsx
│   │       dummy.spec.tsx
│   │
│   └───pageB
│           index.tsx
│
├───pages
│       index.tsx
+|        pageA.tsx
│       pageB.tsx
```
because `http://localhost:8080/pageA` doesn't work. 
If we don´t put it in this way, another solution should be to insert a URL friendly in _server.js_
Would there be a better solution?